ores/applications/wsgi.py

```diff
@@ -0,0 +1,36 @@
+"""
+Runs a uwsgi server.
+
+Usage:
+    wsgi [--config-dir=<path>]... [--logging-config=<path>]
+
+Options:
+    -h --help                Prints this documentation
+    --config-dir=<path>      The path to a directory containing configuration
+                             [default: config/]
+    --logging-config=<path>  The path to a logging configuration file
+"""
+import logging
+
+import docopt
+from ores.wsgi import server
+
+from .util import build_config
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv=argv)
+    run(config_dirs=args['--config-dir'],
+        logging_config=args['--logging-config'])
+
+
+def run(*args, **kwargs):
+    application = build(*args, **kwargs)
+    logging.getLogger('ores').setLevel(logging.DEBUG)
+    application.debug = True
```

Comment:
Quoting from another comment above:
> these applications will be configured via uwsgi or celery worker and will not be called as command-line utilities. See my notes on the initial PR.

The logging *is* configurable, but when executing the application script directly increased logging is very useful as this is only done when debugging.